### PR TITLE
Homework1

### PR DESCRIPTION
**Pull request by oksanazah**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+ */
+app.get('/products', (req, res) => {
+	res.status(200).json({ data: 'Products' });
+});
+
+app.use((req, res, next) => {
+	res.status(404).send('404 Not Found');
+});
+
+app.use((req, res, next) => {
```
Comment (VitalyaIlin) [inline src/index.js:61]: Write single middleware  and handle all types of errors inside it

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+ *       200:
+ *         description: List of products
+ *       500:
+ *         description: Internal Server Error
+ */
+app.get('/products', (req, res) => {
+	res.status(200).json({ data: 'Products' });
+});
+
+app.use((req, res, next) => {
```
Comment (VitalyaIlin) [inline src/index.js:57]: Write single middleware  and handle all types of errors inside it

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+ *     responses:
+ *       200:
+ *         description: List of products
+ *       500:
+ *         description: Internal Server Error
+ */
+app.get('/products', (req, res) => {
+	res.status(200).json({ data: 'Products' });
+});
+
```
Comment (VitalyaIlin) [inline src/index.js:57]: Optional: Add one more endpoint to fetch product by ID and handle 404 error if product is not found

**Review by VitalyaIlin — APPROVED** [review]

File: src/index.js (hunk 3 above)
Comment (oksanazah) [reply src/index.js:57]: added endpoint to fetch product by id and handled 404 error

**Pull request merged**